Packs/F5/Integrations/F5/integration-F5.yml

```diff
@@ -121,28 +157,89 @@ script:
             default:
                 return undefined;
         }
+        if ('source' in args) {
+             args['source'] = formatIPList(args["source"]);
+        }
+        if ('destination' in args) {
+             args['destination'] = formatIPList(args["destination"]);
+        }
         return JSON.stringify(args);
     }
+
     switch (command) {
         case 'test-module':
             if (sendRequest('GET', testUrl)) {
                 return 'ok';
             }
             return 'Gevald!!';
         default:
-            return sendRequest(getMethod(command), getUrl(command, args), buildBody(command, args));
+            body = sendRequest(getMethod(command), getUrl(command, args), buildBody(command, args));
+            md = tableToMarkdown("F5 Results", body);
+            cmd_arr = command.split('-');
+            key = "F5." + cmd_arr[cmd_arr.length - 1];
+            entry = {
+                'Type' : entryTypes.note,
+                'Contents': body,
+                'ContentsFormat' : formats.json,
+                'HumanReadable': md,
+                'ReadableContentsFormat' : formats.markdown
+            };
+            entry['EntryContext'] = {}
+            entry['EntryContext'][key] = body
+
+            return entry;
     }
   type: javascript
   commands:
   - name: f5-create-policy
-    arguments: []
+    arguments:
+    - name: name
+      required: true
+      description: The name of the policy to create
+    - name: description
+      description: The description of the policy
+    - name: copyFrom
+      description: The policy to copy rules from to use as template for new policy
+    - name: partition
+      description: 'The partition (Default: Common)'
     description: Creates an F5 firewall policy.
   - name: f5-create-rule
     arguments:
     - name: policy-name
       required: true
       default: true
       description: The policy name the rule will be associated with.
+    - name: ruleList
+      description: Specifies a list of rules to evaluate. See security firewall rule-list. If a rule-list is specified then only the schedule and status properties effect the rule.
```

Comment:
What is This ruleList?
and what is "See security firewall rule-list"?

Reply:
updated to have a reference to the specific command for listing rules.